server/handlers/default.go

```diff
@@ -245,6 +245,75 @@ func getKeyHandler(ctx context.Context, w http.ResponseWriter, r *http.Request,
 	return nil
 }
 
+// RotateKeyHandler rotates the remote key for the specified role, returning the public key
+func RotateKeyHandler(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+	defer r.Body.Close()
+	vars := mux.Vars(r)
+	return rotateKeyHandler(ctx, w, r, vars)
+}
+
+func rotateKeyHandler(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
```

Comment:
Non-blocking - since this code is basically the same as that for `getKeyHandler`, except for logging and which function `GetTimestampKey` vs `RotateTimestampKey` and `GetSnapshotKey` vs `RotateSnapshotKey`, I was wondering if we could refactor at least the logging part to be the same except for plugging in the method?